MsgPack/MsgPackSerializer.cs

```diff
@@ -105,7 +106,23 @@ public void Serialize(short v)
 				Serialize(unchecked((ushort)v));
 		}
 
-		public void Serialize(ushort v)
+        public void Serialize(Delegate d)
+		{
+            // this works only if msgpack lib is built within fivem, it's not really elegant imho
+            // but it works, so let's not break it now, shall we?
+            // TODO: Make this work without depending from ReferenceFunctionManager
+			// and make our own Canonicalization of the reference id
+			// TODO: Add support for byte 10 (Remote Delegate)? don't think it's enabled in FiveM currently
```

Comment:
What about the todos here?

Reply:
- At the moment msgpack delegate serialization depends on ReferenceFunctionManager which is not in msgpack repository but when linked in FiveM's project builds and works correctly.
- Second todo i wanted to ask if i need to also add code 10 or if code 11 is enough for delegate serialization.
as far as i tested code 11 works fine without the need of adding code 10 in the remote delege preprocessor symbols

Reply:
As long as it builds inside the submodule together inside the fivem folder its fine 👍